### PR TITLE
Added Base of Application and Updated Readme file

### PR DESCRIPTION
**Pull request by wMercadoJala**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+source "https://rubygems.org"
```
Comment (VanessaVargas) [inline Gemfile]: single quotes

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+namespace :db do
+  task :load_config do
+    # TODO add task to load DB
+  end
+end
+
+namespace :devops do
+  desc "Starts the thin web server through rackup."
```
Comment (VanessaVargas) [inline Rakefile]: single quotes

File: app/controllers/list_controller.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+require_relative 'main_controller'
+
+class ListController < MainController
+
+  get '/lists' do
+    "Hello, List Controller!"
```
Comment (VanessaVargas) [inline app/controllers/list_controller.rb]: single quotes

File: app/controllers/main_controller.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+require 'sinatra/reloader'
+
+class MainController < Sinatra::Base
+  register Sinatra::Reloader
+
+  configure do
+    set :public_folder, 'public'
+    set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "password_security"
```
Comment (VanessaVargas) [inline app/controllers/main_controller.rb]: single quotes

File: app/controllers/main_controller.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  get '/' do
+
+    "Hello, ToDo Application!!!"
```
Comment (VanessaVargas) [inline app/controllers/main_controller.rb]: single quotes

File: Gemfile [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
```
Comment (VanessaVargas) [inline Gemfile]: delete comment

**Comment by fercho-ayala** [comment]
Looks good for me.

**Pull request merged**